code/modules/hydroponics/grown_inedible.dm

```diff
@@ -8,9 +8,8 @@
 	var/plantname
 	var/potency = 1
 
-/obj/item/weapon/grown/New(newloc,planttype)
-
-	..()
+/obj/item/weapon/grown/Initialize(mapload, planttype)
+	. = ..()
```

Comment:
Will `planttype` argument be accessable?

Reply:
I see no problem with access, unless someone messes with new and skips loc as first argument writing something like `new type(planttype)` instead of `new type(loc, planttype)` or even using named arguments which are actually dead with Initialization as i know.